deps/rabbitmq_mqtt/src/rabbit_mqtt_processor.erl

```diff
@@ -509,6 +521,17 @@ process_request(?SUBSCRIBE,
                                        reason_codes = lists:reverse(ReasonCodesRev)}},
     _ = send(Reply, State1),
     case hd(ResultRev) of
+        {error, access_refused} ->
+            %% ignore_unauthorized setting:
+            %%  - true: not disconnect client,send retained messages for the successfully subscribed topics.
```

Comment:
"do not disconnect the client", "for the topics to which the client could successfully subscribe"